src/main/java/frc/robot/Constants.java

```diff
@@ -421,416 +416,53 @@ public final class SimulationConstants {
-          kRedStickPoses[11][1],
-          kL3H,
-          new Rotation3d(Degrees.zero(), Degrees.of(-35), Degrees.zero())),
-
-      // Level 4 (L4 - Highest Branches)
-      new Pose3d(
-          kRedStickPoses[0][0],
-          kRedStickPoses[0][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-      new Pose3d(
-          kRedStickPoses[1][0],
-          kRedStickPoses[1][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-      new Pose3d(
-          kRedStickPoses[2][0],
-          kRedStickPoses[2][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-      new Pose3d(
-          kRedStickPoses[3][0],
-          kRedStickPoses[3][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-      new Pose3d(
-          kRedStickPoses[4][0],
-          kRedStickPoses[4][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-      new Pose3d(
-          kRedStickPoses[5][0],
-          kRedStickPoses[5][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-      new Pose3d(
-          kRedStickPoses[6][0],
-          kRedStickPoses[6][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-      new Pose3d(
-          kRedStickPoses[7][0],
-          kRedStickPoses[7][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-      new Pose3d(
-          kRedStickPoses[8][0],
-          kRedStickPoses[8][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-      new Pose3d(
-          kRedStickPoses[9][0],
-          kRedStickPoses[9][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-      new Pose3d(
-          kRedStickPoses[10][0],
-          kRedStickPoses[10][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-      new Pose3d(
-          kRedStickPoses[11][0],
-          kRedStickPoses[11][1],
-          kL4H,
-          new Rotation3d(Degrees.zero(), Degrees.of(90), Degrees.zero())),
-    };
+  public static class SubsystemEnabled {
+    public static final boolean kDrivebase = false || Robot.isSimulation();
+    public static final boolean kVision = false || Robot.isSimulation();
+    public static final boolean kIntake = false || Robot.isSimulation();
+    public static final boolean kCoralScorer = false || Robot.isSimulation();
+    public static final boolean kElevator = false || Robot.isSimulation();
+    public static final boolean kAlgaeRemover = false || Robot.isSimulation();
+    public static final boolean kClimber = false || Robot.isSimulation();
+    public static final boolean kStreamDecks = false || Robot.isSimulation();
```

Comment:
We'll want to do crash testing in simulation, so I don't think the Robot.isSimulation() addition is helpful